apps/api/src/domain/models/audio-visual/audio-item/repositories/arango-audio-item-query-repository.spec.ts

```diff
@@ -279,6 +283,62 @@ describe(`ArangoAudioItemQueryRepository`, () => {
         });
     });
 
+    describe(`tag`, () => {
+        const existingTagLabel = 'plants';
+
+        const existingTag = buildTestInstance(EventSourcedTagViewModel, {
+            id: buildDummyUuid(90),
+            label: existingTagLabel,
+            name: buildMultilingualTextWithSingleItem(existingTagLabel),
+            // TODO do we want this here?
+            members: [],
+        });
+
+        const newTagId = buildDummyUuid(91);
+
+        const newTagLabel = 'animals';
+
+        // TODO use event sourced setup?
+        const newTag = buildTestInstance(Tag, {
+            id: newTagId,
+            label: newTagLabel,
+        });
+
+        const targetTerm = buildTestInstance(EventSourcedAudioItemViewModel, {
+            tags: [existingTag],
+        });
+
+        beforeEach(async () => {
+            await databaseProvider.getDatabaseForCollection(ArangoCollectionId.tags).clear();
```

Comment:
It might be nice to abstract the test setup for creating tags. Eventually, tags will be joined from a `tag__VIEWS` collection. The event consumers should not touch the domain part of the database at all. We should investigate whether we can make it so on this PR, but if not, we should do this refactor fairly soon. We don't want to propagate such patterns as they will make it difficult to separate out the query DB from the command DB and cost us many of the benefits of the pattern.